pkgs/development/compilers/elm/packages/node/node-composition.nix

```diff
@@ -8,7 +8,7 @@ let
   nodeEnv = import ../../../../node-packages/node-env.nix {
     inherit (pkgs) stdenv lib python2 runCommand writeTextFile writeShellScript;
     inherit pkgs nodejs;
-    libtool = if pkgs.stdenv.hostPlatform.isDarwin then pkgs.cctools or pkgs.darwin.cctools else null;
+    libtool = if pkgs.stdenv.isDarwin then pkgs.cctools or pkgs.darwin.cctools else null;
```

Comment:
Why was this changed back?

Reply:
Maybe because we used my mac to run the generation script. Could probably be reverted?

Reply:
Reverted.